src/styles/index.scss

```diff
@@ -1,3 +1,72 @@
+$cell-size: 100px;
+$default-month-length: 31;
+$days-of-week: (
+  'mon': '0',
+  'tue': '1',
+  'wed': '2',
+  'thu': '3',
+  'fri': '4',
+  'sat': '5',
+  'sun': '6',
+);
+$gap: 1px;
+
 body {
   margin: 0;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  min-height: 100vh;
+  font-family: Arial, sans-serif;
+  font-size: 30px;
+}
+
+.calendar {
+  gap: $gap;
+  display: flex;
+  flex-wrap: wrap;
+  width: 7 * ($cell-size + $gap) - $gap;
+
+  &__day {
+    margin: 0;
+    box-sizing: border-box;
+    height: $cell-size;
+    width: $cell-size;
+    background-color: #eee;
+    border: 1px solid #000;
+    display: flex;
+    align-items: center;
+    justify-content: center;
+
+    &:hover {
+      transition-duration: 0.5s;
```

Comment:
The `transition-duration` property should be specified within the element's base styles, not on hover. This ensures that the transition is also smooth when the hover state is removed. You should move `transition-duration: 0.5s;` to the `.calendar__day` block.